modules/microshift-install-system-requirements.adoc

```diff
@@ -8,15 +8,21 @@
 
 The following conditions must be met prior to installing {microshift-short}:
 
-* A compatible version of {op-system-base} or {op-system-ostree}.
+* A compatible version of {op-system-base-full}.For more information, see link:https://docs.redhat.com/en/documentation/red_hat_build_of_microshift/{ocp-version}/html/getting_ready_to_install_microshift/microshift-install-get-ready#get-ready-install-rhde-compatibility-table_microshift-install-get-ready[Compatibility table].
 * AArch64 or x86_64 system architecture.
 * 2 CPU cores.
-* 2 GB RAM for {microshift-short} or 3 GB RAM, required by {op-system-base} for networked-based HTTPs or FTP installations.
+* 2 GB RAM. Installing from the network (UEFI HTTPs or PXE boot) requires 3 GB RAM for {op-system-base}.
 * 10 GB of storage.
 * You have an active {microshift-short} subscription on your Red Hat account. If you do not have a subscription, contact your sales representative for more information.
-* You have a Logical Volume Manager (LVM) Volume Group (VG) with sufficient capacity for the Persistent Volumes (PVs) of your workload.
+* When your workload requires Persistent Volumes (PVs), you have a Logical Volume Manager (LVM) Volume Group (VG) with sufficient free capacity for the workloads.
+[NOTE]
+====
+These requirements are the minimum system requirements for {microshift-short} and {op-system-base-full}. Add the system requirements for the workload you plan to run.
+For example, if an IoT gateway solution requires 4 GB of RAM, your system needs to have at least 2GB for {op-system-base-full} and {microshift-short}, plus 4GB for the workloads. 6GB of RAM is required in total.
+It is recommended to allow for extra capacity for future needs, if you are deploying physical devices in remote locations. If you are uncertain of the RAM required and if the budget permits, use the maximum RAM capacity the device can support.
+====
 
 [IMPORTANT]
```

Comment:
Question - if you're adding a note, and it's not as important as this IMPORTANT admonition, shouldn't the note go under this? Up to you

Reply:
made both as important